### ISSUE TITLE
Add React Hook Form package.json

### ISSUE THREAD

**Issue by trillium** [hackforla/VRMS#1406]
### Overview
We've got a lot of validation tickets coming up, and getting `react-hook-form` added in earlier will be helpful.

Adding this in earlier will unblock us on 
- #1363
- #1367
- #1368
- #1369
- #1370

### Action Items
- @bkmorgan3 has already done the work on #1400, just need to run the following commands for a new pull request:

```
git cherry pick fdecf24
git add client/package.json
git add client/yarn lock
```
